src/util.ts

```diff
@@ -290,3 +294,81 @@ export const randomBytes = function(n) {
   }
   return buf;
 }
+
+/**
+ * Generates a private key for use in maintaining connection to device.
+ * @param {Buffer} deviceId - The device ID of the device you want to generate a token for.
+ * @param {Buffer} password - The password entered when connecting to the device.
+ * @param {Buffer} appName - The name of the application.
+ * @returns a private key as a Buffer
+ */
+export const generatePrivateKey = (
```

Comment:
This name is too generic. Please change to `generateAppSecret` or something.

Reply:
[123bf7d](https://github.com/GridPlus/gridplus-sdk/pull/371/commits/123bf7d5f7a9e03b3851717dcbe3ec1b1fb70c29)